frontend/src/app/note/note.component.spec.ts

```diff
@@ -119,7 +119,7 @@ describe('NoteComponent', () => {
     expect(tester.delete.disabled).toBe(true);
   });
 
-  it('should switch to edit mode and give the focus to the text area', () => {
+  it('should switch to edit mode and give the focus to the text area', async(() => {
```

Comment:
This is probably fixing the jasmine issue because it creates a "context" kept in the promise via the zone.